src/main/java/com/iexec/commons/poco/chain/IexecHubAbstractService.java

```diff
@@ -134,38 +140,19 @@ private static int scoreToWeight(int workerScore) {
         return Math.max(workerScore / 3, 3) - 1;
     }
 
-    /*
-     * We wan't a fresh new instance of IexecHubContract on each call in order to get
-     * the last ContractGasProvider which depends on the gas price of the network
-     */
-    public IexecHubContract getHubContract(ContractGasProvider contractGasProvider) {
-        return getHubContract(contractGasProvider,
-                ChainIdLong.NONE,
-                (int)blockTime.toMillis(),
-                DEFAULT_POLLING_ATTEMPTS_PER_TX_HASH);
-    }
-
-    public IexecHubContract getHubContract(ContractGasProvider contractGasProvider,
-                                           long chainId) {
-        return getHubContract(contractGasProvider,
-                chainId,
-                (int)blockTime.toMillis(),
-                DEFAULT_POLLING_ATTEMPTS_PER_TX_HASH);
-    }
-
     /**
      * Get an IexecHubContract instance.
      *
      * @param contractGasProvider gas provider, useful for sending txs
      * @param chainId chain ID for EIP155 protection
-     * @param watchFrequency frequency for getting tx receipt
      * @param watchAttempts number of attempts to get tx receipt
+     * @param watchFrequency frequency for getting tx receipt
      * @return an IexecHubContract instance
      */
     public IexecHubContract getHubContract(ContractGasProvider contractGasProvider,
```

Comment:
Could this now be private?

Reply:
sure, already done in 32a27d9. We're sync'ed on this one.